app/views/shared/summary/_expenses.html.haml

```diff
@@ -1,3 +1,16 @@
+.error-summary{"aria-labelledby" => "error-summary-expenses", :role => "alert", :tabindex => "-1"}
+  %h1#error-summary-heading-example-1.heading-medium.error-summary-heading
+    There are errors with the travel expenses
+  %p The distance claimed by the provider is greater than the calculated distance for the following expenses:
```

Comment:
unless we want a more specific message i think `greater` will have to be `different`

Reply:
@jsugarman  - I will raise it in JIRA.. thought that as well when I read the line